_acs-aem-commons/features/sling-model-injectors/shared-value-map-value/index.md

```diff
@@ -10,3 +10,30 @@ initial-release: 3.17.0
 ## Purpose
 
 Provides convenience for for accessing Shared Properties as implemented by ACS Commons [Shared Component Properties](/acs-aem-commons/features/shared-component-properties/index.html).
+
+## Example
+
+    @Model(adaptables = {SlingHttpServletRequest.class, Resource.class})
+    public final class SharedComponentPropsExampleModel {
+
+        @SharedValueMapValue(name = "titleText", type = SharedComponentProperties.ValueTypes.SHARED)
+        private String shared;
+
+        @SharedValueMapValue(name = "titleText", type = SharedComponentProperties.ValueTypes.GLOBAL)
+        private String global;
+
+        @SharedValueMapValue(name = "titleText", type = SharedComponentProperties.ValueTypes.MERGED)
```

Comment:
Should we add a comment here that `MERGED` type is optional/default?

Reply:
We could, if you want it to - just say the word.

Reply:
I think it's worth doing so.  This documentation could lead someone to believe this injection is much more complex than it really is.  In practice, an injection is almost always as simple as:
```
@SharedValueMapValue
private String myProp;
```
`name` can almost always default to the java var name, and `type` of MERGED works for almost all use cases.